index.html

```diff
@@ -211,6 +211,33 @@ <h3 class="about__skills__top__title fw-500 fz-20 lh-24 clr-n-900">Skills</h3>
 			</li>
 		</ul>
 	  </section>
+	  <section class="contact">
+		<div class="contact__container">
+			<h2 class="contact__title fw-700 fz-40 lh-52">Contact me</h2>
+			<p class="contact__dsc fw-500 fz-20 lh-28">
+				If you have an application you are interested in developing,
+				a feature that you need built or a project that needs
+				coding. I’d love to help with it
+			</p>
+		  
+		  <form action="https://formspree.io/f/xqkwbzdb" method="post">
+			  <ul class="ul">
+				  <li>
+					  <input type="text" class="contact__name fw-500 fz-17 lh-20" name="contact_name" placeholder="Enter your name" required maxlength="30">
+				  </li>
+				  <li>
+					<input type="email" class="contact__email fw-500 fz-17 lh-20" placeholder="Enter your email" name="contact_email" required>
+				</li>
+				<li>
+					<textarea class="fw-500 fz-17 lh-20" name="contact_message" placeholder="Write your message here" cols="30" rows="8" maxlength="500" required></textarea>
+				</li>
+				<li>
+					<input type="submit" value="Get It Touch" class="btn fw-500 fz-17 lh-24">
+				</li>
```

Comment:
- [ ] [OPTIONAL] Kindly change this button from saying 'Get It Touch' to 'Get In Touch', it's a typo in the Figma design.

Reply:
Thanks immensely for the uplifting review! Will implement the optional suggestion.

Reply:
Done.

Reply:
Thank you.